meta-ostro/recipes-swupd/swupd-client/swupd-client/efi-combo-trigger.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Updates the EFI combo application
+BindsTo=update-triggers.target
+
+[Install]
+WantedBy=update-triggers.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/efi_combo_updater
```

Comment:
On 7 April 2016 at 10:01, Gustavo Sverzut Barbieri <notifications@github.com
> wrote:

> In
> meta-ostro/recipes-swupd/swupd-client/swupd-client/efi-combo-trigger.service
> <https://github.com/ostroproject/meta-ostro/pull/95#discussion_r58907078>
> :
>
​[...]​

> > +ExecStart=/usr/bin/efi_combo_updater
>
> should this tool be in hidden namespace so it doesn't end in $PATH?
> @igor-stoppa <https://github.com/igor-stoppa>
>
> also missing newline :-)
>
​yeah, however I'm not happy of the overall implementation
I have a temporary patch that turns this into a shell script, which is
cleaner
and doesn't rely on mount, however what I really want is to agree with
swupd guys
to introduce the concept of generic update-boot-files

I had some exchange with Arjan but the discussion kinda died out.
Your comments there would be appreciated.​



-- 
igor